Make transparency of folders configurable

FliegendeWurst opened the pull request:
#### What is it?
- [x] Feature

#### Description of the changes in your PR
This PR makes the transparency of folders configurable. In my opinion the default is too opaque.

#### Before/After Screenshots/Screen Record
![Screenshot_Transparency](https://github.com/user-attachments/assets/95f21bcc-984b-4ec0-bde0-31be02facc04)
- Before (left): standard 10% transparency
- After (right): 90% transparency

#### Alternatives
Make the default >70% transparency and do not add a configuration option.

#### Acknowledgement
- [x] I read the [contribution guidelines](https://github.com/FossifyOrg/Launcher/blob/main/CONTRIBUTING.md).

Aga-C commented:
Thanks for contributing, but next time please create an issue or discussion first (as stated in the contribution guidelines).

naveensingh commented:
I found a couple issues:

 - The whole preference option is not clickable and there is no ripple feedback.
 - The change isn't applied immediately. I have to restart the app for changes to take effect.
 - The preference should be located under the 'Home Screen' section. We don't have folders anywhere else.
 - The current name 'Transparency of folder background' is a bit wordy and works differently compared to other launcher apps. Let's rename it to 'Folder background opacity' where 0% means fully transparent and 100% means fully opaque.

FliegendeWurst commented:
Okay with me. I have since switched to a darker background where the current opacity looks good enough.

I'll leave my branch up, in case someone else wants this feature and addresses the feedback above.
